OpenTabletDriver.Web/Views/Wiki/FAQ/Windows.cshtml

```diff
@@ -77,4 +77,16 @@
         If this does not solve the issue, check that you are not using the "Administrator" named account as it overrides User Account
         Control.
     </p>
+</div>
+
+<hr/>
+
+<h3 class="wiki-nav-item pb-2" id="rtss">
+    Why do i get a blank screen when opening OpenTabletDriver?
```

Comment:
```suggestion
    Why do I get a blank screen when opening OpenTabletDriver?
```

Reply:
It's also not blank, but crashes completely, and then closes after some time.